pkg/api/resource_helpers.go

```diff
@@ -97,3 +97,26 @@ func IsNodeReady(node *Node) bool {
 	}
 	return false
 }
+
+// PodRequestsAndLimits returns a dictionary of all defined resources summed up for all
+// containers of the pod.
+func PodRequestsAndLimits(pod *Pod) (reqs map[ResourceName]resource.Quantity, limits map[ResourceName]resource.Quantity, err error) {
```

Comment:
if this stayed in `pkg/kubectl/describe.go` tho that'd be two less non-mesos files we're modifying, which might make it easier to merge.